build/README.md

```diff
@@ -12,37 +12,28 @@ Trigger a build on Netlify, if there's no site for this repo it can automagicall
 - `NETLIFY_CMD` - *Optional* Build command to build site
 - `NETLIFY_DIR` - *Optional* The directory to publish (relative to root of your repo)
 
-## Examples
-
-Trigger a build to a specific site in Netlify
-
-```hcl
-workflow "Publish on Netlify" {
-  on = "push"
-  resolves = ["Publish"]
-}
-
-action "Publish" {
-  uses = "netlify/actions/build@master"
-  secrets = ["GITHUB_TOKEN", "NETLIFY_SITE_ID"]
-}
-```
```

Comment:
What's the reason to remove this example?

Reply:
@kitop 
HCL syntax for GitHub Actions is already deprecated.
That's why I thought this syntax should be replaced with YAML syntax.

But, if there are people who want to see HCL syntax, this syntax are should not removed.
How about you?

Reply:
Sorry, my questions was regarding that there's no YAML example for this. It was removed, but didn't get a yaml version like the rest. Is that on purpose?

Reply:
Oops. It's my mistake again.

Reply:
@kitop 
Done :)